### PR TITLE
Support switch languages in the context menu #83

### PR DESCRIPTION
**Pull request by lingkeyang**

### CODE REVIEW

File: i18n/locales/en_US.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "en_US"
+desc: "English"
+tray:
+  unknown: "UnKnown"
+  app:
+    title_not_login: "Need Login into PairMesh"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:20]: Please log in to PairMesh

File: i18n/locales/en_US.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    title_disconnected: "Network disconnected, please click to reconnect"
+    title_connecting: "Connecting..."
+  auth_expires_in:
+    title_foramt: "Certification will expires in %d hours"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:24]: Certificate expires in %d hours

File: i18n/locales/en_US.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    tips: "Certificate Expiration Reminder"
+  my_address: "My address: %s"
+  connect:
+    title: "Online"
+    tips: "Online Current Node"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:29]: Current Online Nodes

File: i18n/locales/en_US.yaml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  disconnect:
+    title: "Offline"
+    tips: "Offline Current Node"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:32]: Current Offline Nodes

File: i18n/locales/en_US.yaml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  admin:
+    title: "Console"
+    tips: "Login into PairMesh Console"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:35]: Login to PairMesh Console

File: i18n/locales/en_US.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+  login: "Login"
+  profile:
+    console: "Management Console"
+    logout: "Logout"
+  device: "Current Device\t%s"
+  my_devices: "My Device"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:41]: My Devices

File: i18n/locales/en_US.yaml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+  my_networks: "My Network"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:42]: My Networkds

**Review by vodkaslime — CHANGES_REQUESTED** [review]
Nice work! Commented on the en_us translation part.

File: i18n/locales/en_US.yaml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+  admin:
+    title: "Console"
+    tips: "Login to PairMesh Console"
+  login: "Login"
+  profile:
+    console: "Management Console"
+    logout: "Logout"
+  device: "Current Device\t%s"
+  my_devices: "My Devices"
+  my_networks: "My Networkds"
```
Comment (vodkaslime) [inline i18n/locales/en_US.yaml:42]: Networks

**Review by vodkaslime — APPROVED** [review]
Well done, LGTM

**Review by hi-windy — APPROVED** [review]

**Comment by hi-windy** [comment]
Nice work!

@lingkeyang Thanks for your contribution.

**Pull request merged**